imports/plugins/included/authnet/server/methods/authnet.js

```diff
@@ -1,13 +1,15 @@
 /* eslint camelcase: 0 */
 /* eslint quote-props: 0 */
 // meteor modules
+import accounting from "accounting-js";
 import { Meteor } from "meteor/meteor";
 import { check, Match } from "meteor/check";
 import { Promise } from "meteor/promise";
 
 import AuthNetAPI from "authorize-net";
 import { Reaction, Logger } from "/server/api";
 import { Packages } from "/lib/collections";
+import { PaymentMethod } from "/lib/collections/schemas";
```

Comment:
curious why we aren't using the plugin specific schema?

import { AuthNetPayment } from "../../lib/collections/schemas";